schema.sql

```diff
@@ -40,4 +40,31 @@ ALTER TABLE ANIMALS ADD COLUMN OWNERS_ID int
 ALTER TABLE ANIMALS ADD CONSTRAINT CONSTRAINT_FKO
 FOREIGN KEY (OWNERS_ID) REFERENCES OWNERS(ID) 
 ON DELETE CASCADE 
-ON UPDATE CASCADE
\ No newline at end of file
+ON UPDATE CASCADE
+
+-- Create a table named vets with the following columns:
+
+CREATE TABLE VETS (
+ID SERIAL PRIMARY KEY NOT NULL,
+NAME VARCHAR(100) NOT NULL,
+AGE INT NOT NULL,
+DATE_OF_GRADUATION DATE NOT NULL
+)
+
+CREATE TABLE specializations (
+  VID INT NOT NULL,
+  SID INT NOT NULL,
```

Comment:
- [ ] Please use descriptive  names for your table columns. `VID` & `SID` aren't descriptive.  Why not use `species_id` and `vet_id` ?